src/libraries/Native/Unix/System.Security.Cryptography.Native/android_security.h

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#pragma once
+
+#include "pal_compiler.h"
+
+#define SUCCESS 1
+
+PALEXPORT int32_t CryptoNative_GetRandomBytes(uint8_t* buf, int32_t num);
```

Comment:
Where are these functions used?  Or is this reusing the declaration from [`Interop.RAND.cs`](https://github.com/dotnet/runtime/blob/8713a04c225c00c4c4b00dc9ae4b904c62c522f6/src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.RAND.cs)?

Reply:
Yep, called from C#